core/controllers/questions_list_test.py

```diff
@@ -183,33 +185,36 @@ def test_get_question_count_succeeds(self):
             self.admin_id, question_id_1, self.skill_id_2, 0.3)
 
         json_response = self.get_json(
-            '%s/%s,%s' % (
+            '%s/%s' % (
                 feconf.QUESTION_COUNT_URL_PREFIX,
-                self.skill_id, self.skill_id_2
+                json.dumps('%s,%s' % (self.skill_id, self.skill_id_2))
```

Comment:
Hmm, so what is the final URL generated here? This controller is called from the frontend end as well and if teh expected URL in the controller is different, the frontend should also be updated.